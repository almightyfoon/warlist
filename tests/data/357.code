c1501beS9g9g9A9u4j70ayaccW9U___-0n8U9cc09Aakanaj9be59Cbz