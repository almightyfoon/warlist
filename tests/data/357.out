https://conflictchamber.com/#c1501beS9g9g9A9u4j70ayaccW9U___-0n8U9cc09Aakanaj9be59Cbz

Cygnar Army - 75 / 75 points

[Haley 3] Major Prime Victoria Haley [+25]
 - Grenadier [9]
 - Grenadier [9]
 - Thorn [13]
Ragman [4]
Viktor Pendrake [4]
Captain Sam MacHorne & the Devil Dogs (max) [18]
 - Captain Jonas Murdoch [4]
Storm Lances (min) [12]
Tempest Blazers (min) [11]
Trencher Infantry (max) [16]

Cygnar Army - 79 / 75 points
[Theme] Heavy Metal

!!! This army must contain 75 points or fewer

[Haley 2] Major Victoria Haley [+25]
 - Firefly [8]
 - Stormwall [39]
 - Thorn [13]
 - Squire [0(5)]
Captain Arlan Strangewayes [0(4)]
Journeyman Warcaster [0(4)]
 - Charger [9]
Lieutenant Allison Jakes [4]
 - Cyclone [13]
Storm Strider [18]
